### PR TITLE
small fixes & improvements

### PR DESCRIPTION
**Pull request by TC-MO**
removed all unnecessary double spaces & trailing spaces
small rephrasings for better clarity
typo fixes

### CODE REVIEW

**Comment by TC-MO** [comment]
also the link to `live-view-web-server` seems to be leading to nowhere?

File: README.md [hunk 1]
```diff
@@ -165,16 +162,14 @@ can set a price tag for the usage of their Actors, and thus earn income
...
 ## Basic concepts
 
 This section describes core features of Actors, what they are good for,
-and how Actors differ from other serverless computing systems. 
+and how Actors differ from other serverless computing systems.
 
 ### Input
 
-Each Actor accepts an **input object**, which tells it what it should do.
+Each Actor accepts an **input object**, which specifies its operation parameters.
```
Comment (jancurn) [inline README.md:172]: I'd keep the old version, it's more specific

File: README.md [hunk 2]
```diff
@@ -1252,7 +1238,7 @@ or to run an arbitrary web app.
 The port on which the Actor can launch the web server
 is specified by the `ACTOR_WEB_SERVER_PORT` environment variable.
 
-The web server is started, it is exposed to the public internet on a **live view URL** identified 
+The web server is started, it is exposed to the public internet on a **live view URL** identified
```
Comment (jancurn) [inline README.md:1241]: This should be: Once the web server is started ...

**Review by jancurn — APPROVED** [review]
Great, thanks for the polishing

File: README.md [hunk 3]
```diff
@@ -15,62 +15,119 @@ By [Jan Čurn](https://apify.com/jancurn),
...
-  * [Backward compatibility](#backward-compatibility)
-- [Development](#development)
-  * [Local development](#local-development)
-  * [Deployment to Apify platform](#deployment-to-apify-platform)
-  * [Continuous integration and delivery](#continuous-integration-and-delivery)
-  * [Actorizing existing code](#actorizing-existing-code)
-- [Sharing and publishing](#sharing-and-publishing)
-  * [Monetization](#monetization)
-- [Future work](#future-work)
+- [The Web Actor Programming Model Whitepaper \[DRAFT\]](#the-web-actor-programming-model-whitepaper-draft)
```
Comment (jancurn) [inline README.md:18]: Please use `npm run build-toc` for this, to limit the dept

**Review by jancurn — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (TC-MO) [reply README.md]: fixed, one of my `.md` extension autogenerated toc, and I did not notice it

**Comment by jancurn** [comment]
@TC-MO thanks!

**Pull request merged**